src/house.cpp

```diff
@@ -499,52 +502,11 @@ void AccessList::addGuildRank(const std::string& name, const std::string& rankNa
 	}
 }
 
-void AccessList::addExpression(const std::string& expression)
-{
-	if (std::find(expressionList.begin(), expressionList.end(), expression) != expressionList.end()) {
-		return;
-	}
-
-	std::string outExp;
-	outExp.reserve(expression.length());
-
-	std::string metachars = ".[{}()\\+|^$";
-	for (const char c : expression) {
-		if (metachars.find(c) != std::string::npos) {
-			outExp.push_back('\\');
-		}
-		outExp.push_back(c);
-	}
-
-	replaceString(outExp, "*", ".*");
-	replaceString(outExp, "?", ".?");
-
-	try {
-		if (!outExp.empty()) {
-			expressionList.push_back(outExp);
-
-			if (outExp.front() == '!') {
-				if (outExp.length() > 1) {
-					regExList.emplace_front(std::regex(outExp.substr(1)), false);
-				}
-			} else {
-				regExList.emplace_back(std::regex(outExp), true);
-			}
-		}
-	} catch (...) {}
-}
-
 bool AccessList::isInList(const Player* player)
 {
 	std::string name = asLowerCaseString(player->getName());
 	std::cmatch what;
```

Comment:
you can remove these variables `name` and `what`, they are no longer being used.

Reply:
..and you need to reset the variable `allowEveryone`
```diff
void AccessList::parseList(const std::string& list)
{
	playerList.clear();
	guildRankList.clear();
+	allowEveryone = false;
```
Why? because even if you remove `*` from aleta sio/som, players continue to enter the house.

Reply:
good catch, thanks!